vdev/src/git.rs

```diff
@@ -9,6 +9,26 @@ pub fn current_branch() -> Result<String> {
     Ok(output.trim_end().to_string())
 }
 
+pub fn checkout_branch(branch_name: &str) -> Result<()> {
+    let _output = capture_output(&["checkout", "-b", branch_name])?;
+    Ok(())
+}
+
+pub fn merge_branch(branch_name: &str) -> Result<()> {
+    let _output = capture_output(&["merge", "--ff", branch_name])?;
+    Ok(())
+}
+
+pub fn tag_version(version: &str) -> Result<()> {
+    let _output = capture_output(&["tag", "-a", version, "-m", version])?;
```

Comment:
Prefer using long options where available to spell out what they stand for:
```suggestion
    let _output = capture_output(&["tag", "--annotate", version, "--message", version])?;
```

Reply:
I have added these changes and will push